doc/interop-test-descriptions.md

```diff
@@ -887,6 +943,19 @@ payload body of size `SimpleRequest.response_size` bytes and type as appropriate
 for the `SimpleRequest.response_type`. If the server does not support the
 `response_type`, then it should fail the RPC with `INVALID_ARGUMENT`.
 
+### CompressedResponse
+[CompressedResponse]: #compressedresponse
+
+When the client sets `SimpleRequest.request_compressed_response` to true, the
+response is sent back compressed.
+
+### CompressedRequest
```

Comment:
the discussed three-steps procedure (expect-compressed, expect-uncompressed, lie-and-see-if-server-notices) is described in the client_compressed_{unary,streaming} testcases